client/chat/channel.jsx

```diff
@@ -81,6 +81,13 @@ const UserListEntryItem = styled.li`
     cursor: pointer;
     background-color: rgba(255, 255, 255, 0.08);
   }
+
+  ${props => {
+    if (props.isOverlayOpen) {
+      return `background-color: ${grey800};`
```

Comment:
I think ideally this would be the same color as the hover state, instead of a unique value